jax-rs-linker-integration-tests/xml-webapp/pom.xml

```diff
@@ -48,11 +48,16 @@
         <dependency>
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
+            <scope>provided</scope>
```

Comment:
Je ne connais pas du tout la lib et la façon dont elle est utilisée chez nous, mais je préfère quand même poser la question : Comment tu détermines quelles libs doivent être `provided` par le projet appelant ou non ?

je vois du `jersey` en scope `compile` , du jaxRS en scope `provided`...

En gros je voudrais m'assurer que l'on ait le moins de dépendances en scope `compile` possible :innocent:

Reply:
C'est un module de tests d'intégrations, ça simule une vraie application dans laquelle tu utiliserais `jersey` (d'où le scope compile).  

Le scope `provided` de `javax.ws.rs-api` vient du fait que Jersey embarque déjà la spec dans `jakarta.ws.rs-api`, c'est le nouveau nom de l'artifact depuis le renommage Java EE. Du coup il y a des warning au démarrage du Jetty. On pourrait enlever `javax.ws.rs-api` dans les modules qui servent aux tests d'intégrations.

Reply:
Je vois ! 

Dans ce cas, puisque cela ne sert qu'aux tests d'intégrations, je pense que les dépendances 

`jersey-container-servlet` et `jersey-hk2` peuvent être déclarés en scope `runtime` ?

soit 1 fois dans le `<dependencyManagement>` soit 3 fois dans les 2 modules d' IT 

J'ai testé en local, les tests passent.

Par ailleurs, puisque ces dépendances sont spécifiques au module d'IT, on pourrait imaginer les déplacer du `pom` principal vers le `pom` de `integration-tests` ?

Mais je suis en train de me dire qu'on commence à sortir du cadre de cette PR :fearful: